__tests__/__snapshots__/check-fixtures.js.snap

```diff
@@ -381,24 +381,6 @@ import { unusedEs6Import } from './stub/unusedEs6Import';
       },
     ],
   },
-  {
-    "column": 1,
-    "endColumn": 14,
-    "endLine": 8,
-    "fix": {
-      "range": [
-        192,
-        205,
-      ],
-      "text": "// @ts-expect-error",
-    },
-    "line": 8,
-    "message": "Use "@ts-expect-error" to ensure an error is actually being suppressed.",
-    "messageId": "preferExpectErrorComment",
-    "nodeType": "Line",
-    "ruleId": "@typescript-eslint/prefer-ts-expect-error",
-    "severity": 2,
-  },
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/pull/9081

The `@typescript-eslint/prefer-ts-expect-error` rule is deprecated; the bug is caught by `@typescript-eslint/ban-ts-comment` (line 360 of the snapshot), so we are good.

Ref: https://typescript-eslint.io/rules/prefer-ts-expect-error/